services/dy-3dvis/simcoreparaviewweb/Dockerfile

```diff
@@ -54,6 +54,7 @@ RUN curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/py
 # install simcore-sdk
 RUN export PATH="${PYENV_ROOT}/bin:$PATH" && \
     eval "$(pyenv init -)" && \
+    pip install git+https://github.com/ITISFoundation/osparc-simcore.git@master#subdirectory=packages/service-library &&\
```

Comment:
should's we pin these to a given tag instead of master ??
For all packages we could set bumpversion to create a tag for us (in setup.cfg , just turn tag=True). At that point
```cmd
cd packages/service-library
make version-patch
```
will produce the appropriate tag (e.g. ``packages-service-library-v0.3.2``), then
```
pip install git+https://github.com/ITISFoundation/osparc-simcore.git@packages-service-library-v0.3.2#subdirectory=packages/service-library 
```
should do it

Reply:
yep we definitely need that to prevent the issue next time.